server/routerlicious/packages/memory-orderer/src/localNode.ts

```diff
@@ -173,6 +173,13 @@ export class LocalNode extends EventEmitter implements IConcreteNode {
             const socket = new Socket<INodeMessage>(wsSocket);
             const subscriber = new RemoteSubscriber(socket);
 
+            // server side listening for pong events.
+            socket.on("pong", (cb: any) => {
```

Comment:
This is a duplicate handler to the one added in services-shared/src/socketIoServer.ts